composer.json

```diff
@@ -11,17 +11,43 @@
   ],
   "minimum-stability": "dev",
   "require": {
-    "php": ">=7.2",
-    "appsero/client": "dev-develop",
+    "php": ">=7.4",
+    "appsero/updater": "^v1.0.0",
```

Comment:
We do not need ` "appsero/updater":` here. Please remove it from the dependency.